go/pkg/tool/embeddedtool.go

```diff
@@ -157,3 +159,13 @@ func getPathFlag() string {
 	}
 	return pathPrefix
 }
+
+func getSymlinkBehaviorType() string {
+  switch symlinkBehavior {
+  case "preserve", "resolve", "unspecified":
+    return symlinkBehavior
+  }
+
+  log.Exitf("--symlink_behavior must be specified.")
```

Comment:
If you want to validate this, it should be added to [validateFlags](https://github.com/bazelbuild/remote-apis-sdks/blob/719a5dd43ab6b3f125211b216c48c1e8ec1b4354/go/pkg/tool/embeddedtool.go#L141) instead.

Reply:
Will do!

Reply:
Decided against validating and allowing anything but the following to result in `UnspecifiedSymlinkBehavior`:
- `resolve`
- `ResolveSymlink`
- `preserve`
- `PreserveSymlink`